public/app/core/utils/kbn.ts

```diff
@@ -200,9 +200,22 @@ kbn.calculateInterval = (range: TimeRange, resolution: number, lowLimitInterval:
 };
 
 kbn.describe_interval = (str: string) => {
+  // Default to seconds if no unit is provided
+  if (Number(str)) {
+    return {
+      sec: kbn.intervals_in_seconds.s,
+      type: 's',
+      count: parseInt(str, 10),
```

Comment:
Don't we need floating point here? Not sure about the `interval_regex` seems like it allows floating point but then we also use `parseInt`, but just wanted to check to be sure.

Reply:
That's true. I was going by the `parseInt` that was already there. I suppose those two should be harmonized - either the regex changed or using `parseFloat` instead.

So - do we want to allow the user to be able to enter floating point values?

Reply:
Well I just wanted to make sure this is the same but I did not run it so I was not sure. It should definitely be unified if it isn't at the moment, probably in alignment with the old behavior. So if you can specify `0.01s` we should also allow `0.01` unitless input and treat is as `0.01s`.

Reply:
But the old behavior already conflicted. I didn't change either the regex, nor the original `parseInt`.
The part I added works precisely like the original code.
If you enter `1.5m` in our cluster right now, you'll see that it results in step `60`, even though it should obviously be `90` if floating point was working.

Maybe it was intentional to permit floating point values in principle but to then silently drop the part after the decimal point?

Reply:
Just spoke about this with @aocenas , we agreed that the previous behavior will remain untouched and I should just ensure that when numbers without units are entered, it works the same way. Which it does, so this is good to go. :rocket: